api/v1alpha1/nodehealthcheck_types.go

```diff
@@ -123,6 +139,20 @@ type NodeHealthCheckStatus struct {
 	// +listMapKey=type
 	// +optional
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="phase",xDescriptors="urn:alm:descriptor:com.tectonic.ui:text"
```

Comment:
> displayName="phase",xDescriptors="urn:alm:descriptor:com.tectonic.ui:text"

Is that for the UI?

Reply:
yes: https://github.com/openshift/console/tree/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors#operator-descriptors